js/stat.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var CLOUD_WIDTH = 420;
+var CLOUD_HEIGHT = 270;
+var CLOUD_X = 100;
+var CLOUD_Y = 10;
+var GAP = 20;
+var FONT_GAP = 15;
+var MAX_BAR_HEIGHT = 150;
+var BAR_WIDTH = 40;
+var BAR_GAP = 50;
+
+var renderCloud = function (ctx, x, y, color) {
+  ctx.fillStyle = color;
+  ctx.fillRect(x, y, CLOUD_WIDTH, CLOUD_HEIGHT);
+};
+
+var getMaxElement = function (arr) {
+  var maxElement = arr[0];
+
+  for (var i = 0; i < arr.length; i++) {
+    if (arr[i] > maxElement) {
+      maxElement = arr[i];
+    }
+  }
+
+  return maxElement;
+};
+
+var getRandomOpacity = function (min, max) {
```

Comment:
```suggestion
var getRandomNumberFromRange = function (min, max) {
```

Переименуй функцию, она не только к опасити отностися